tests/meson.build

```diff
@@ -96,7 +97,8 @@ if gtest_dep.found()
       install_dir: unittest_install_dir
     )
 
-    test('unittest_plugins', unittest_plugins, args: ['--gst-plugin-path=..'])
+    test('unittest_plugins', unittest_plugins, args: ['--gst-plugin-path=..'],
```

Comment:
I think `--gst-plugin-path=..` is no longer needed.

Reply:
@zhoonit Right.